src/pages/hegel/reference/becoming/sublation.mdx

```diff
@@ -0,0 +1,205 @@
+something else&mdash;not something _completely_ different (the `being` of
+`existence` still shares something with `pure being`, its ancestor, so to
+speak), but also not something _entirely_ identical.
+
+The new sublated state refers back to its absolute being, and refers to it as
+negated, whereby it is simultaneously retained. It is this intense binding that
+grants something to be seen _in its truth_: a primordial state that was
+internally incoherent whose incoherence reforged it to this current state.
+
+In this way, _truth_, in light of Hegel's thinking, is never something immediate
+but the result of a development. It is a context made up of sublated moments,
+and whose conception understands that these moments were at one point totally
+unbounded, in their boundlessness utterly failed, and found themselves renewed
+in this multi-dimensional sphere as self-bounded moments.
+
+## Further Commentary
+
+### Houlgate
+
+Stephen Houlgate reads sublation as fusing two processes into one. The first
+process involves negating something while the second process involves preserving
+something. But what is specifically negated is the "simple _immediacy_ &ndash;
+or purity &ndash; of something" (Houlgate 2022, 153). For that reason, since
+what is sublated is no longer purely and immediately itself, Houlgate writes
+that "it must be thought together with its negation" (Houlgate 2022, 153). In
+this respect, it matches what Hegel writes: "something is sublated only in so
+far as it has entered into unity with its opposite" (Hegel 2010, 82/21.95).
+
+Houlgate points out that while `being` and `nothing` are sublated in `becoming`,
+this sublation is strictly incomplete. The sublation here is only partial
+because "each vanishes into its _pure other_, and their immediacy, or purity, is
+thus not thoroughly negated" (Houlgate 2022, 153). Viewed differently, the
+immediacy of each vanishes so completely that there is no remnant which could be
+considered a moment. By contrast, in `determinate being` (`existence`) the
+immediacy is thoroughly negated and "each is present only as united with its
+opposite" (Houlgate 2022, 153).
+
+Houlgate notes further three things about sublation. In the first place,
+sublation is not an operation performed by us&mdash;human thinkers in
+time&mdash;on concepts or determinations of being. "It occurs", Houlgate states,
+"when we render explicit what a category is implicitly, but it is made necessary
+by the category itself" (Houlgate 2022, 153). Categories are thus not sublated
+_by us_, but as the case of `being` and `nothing` shows:
+
+> They do not sublate themselves reciprocally &ndash; the one sublating the
+> other externally &ndash; but each rather sublates itself in itself and is
+> within it the opposite of itself (Hegel 2010, 81/21.93).
+
+In the second place, sublation does not presuppose a unity into which categories
+are sublated (or sublates themselves), but is, "the process through which such a
```

Comment:
"(or sublates themselves) - does the "sublates" refer to categories or sublation? If former, then it should be sublate, if latter, then it's fine."